CHANGELOG.md

```diff
@@ -0,0 +1,27 @@
+# Changelog
+
+## 1.1.3 (2023-12-04)
+
+Full Changelog: [v1.1.3]()
+
+### Bug Fixes
+
+- **sdk:** Updated `add_prompt` to rename prompt name if a different name is passed in ([a29d5f87](https://github.com/lastmile-ai/aiconfig/pull/303/commits/a29d5f87198b42a46335c4f20820f02b7c3b1abf))
+- **typescript-sdk:** Updated Metadata Field to be optional ([cb5fdc5](https://github.com/lastmile-ai/aiconfig/pull/320/commits/cb5fdc51a4c6797352c8c6e8b6eb6deb9fdde32b))
+
+### Feature
```

Comment:
This section should come before bug fixes

Reply:
+1